client/mutation/CreateFeatureMutation.js

```diff
@@ -0,0 +1,42 @@
+import Relay from 'react-relay';
+
+class CreateFeatureMutation extends Relay.Mutation {
```

Comment:
This file must be placed under the `client/components/Feature` as `AddFeatureMutation.js`.